ref/general/auth-filter.adoc

```diff
@@ -15,11 +15,11 @@
 
 You can use authentication filters to specify which authentication mechanism Open Liberty uses to process different HTTP servlet requests.
 
-An authentication filter sets certain criteria that determine how a request for resources is authenticated. For example, if two applications are running on the same Open Liberty server, an authentication filter can specify different types of authentication for each application. In this case, a web application can be secured with the Social Media Login feature, while an associated microservice on the same server is secured with the MicroProfile JSON Web Token feature.
+An authentication filter sets certain criteria that determine how a request for resources is authenticated. For example, if two applications are running on the same Open Liberty server, an authentication filter can specify different types of authentication for each application. In this case, a web application can be secured with a social media login, while an associated microservice on the same server is secured by authenticating to a lightweight directory access protocol (LDAP) registry.
 
 Similarly, in a single application where different kinds of protected resources require different types of authentication, filters can control how authentication is managed for each resource. For example, in a banking application, resources for customers might be accessible through social media login while resources for administrators are secured by SPNEGO authentication.
 
-Authentication mechanisms in Open Liberty are managed by the feature configuration. To specify an authentication filter, configure link:/docs/ref/config/#authFilter.html[the `authFilter` element] in your `server.xml` file. Then, specify a reference to the `authFilter id` attribute within the feature that manages the designated authentication mechanism by using the `authFilterRef` attribute.
+Authentication mechanisms in Open Liberty are managed by the server configuration. In the server.xml file, you can specify filters to control which authentication mechanism is used for different types of requests. For example, you can filter requests that contain the `/mywebapp` URL pattern to be processed by social media login, and requests that contain the `/EmployeeRoleServlet` URL pattern to be processed by SPNEGO. You can configure filters to look for different characteristics of a request, such as URL pattern, IP address, domain name, browser type, and more.
```

Comment:
Should there be backticks on server.xml?

Reply:
yes- thanks